xbmc/addons/kodi-addon-dev-kit/include/kodi/addon-instance/Inputstream.h

```diff
@@ -175,8 +259,14 @@ extern "C" {
 
     // new in API version 2.0.8
     unsigned int m_codecFourCC;          /*!< @brief Codec If available, the fourcc code codec */
-    INPUTSTREAM_COLORSPACE m_colorSpace; /*!< @brief definition of colorspace */
-    INPUTSTREAM_COLORRANGE m_colorRange; /*!< @brief color range if available */
+    COLORSPACE m_colorSpace;             /*!< @brief definition of colorspace */
+    COLORRANGE m_colorRange;             /*!< @brief color range if available */
```

Comment:
remove the vertical alignment, https://github.com/xbmc/xbmc/blob/master/docs/CODE_GUIDELINES.md#35-vertical-alignment